### ISSUE TITLE
Unable to start NVDA portable or from source

### ISSUE THREAD

**Issue by aaclause** [nvaccess/nvda#13406]
### Steps to reproduce:
Unfortunately I'm unable to reproduce this issue on another machine. This only happens on my main computer.
So when I try to launch a copy of NVDA from sources or a portable copy, startup fails. 
Also builds don't fail (i.e. `scons source`, `scons launcher` or `scons dist` work).

### Actual behavior:
Windows error sound is played and NVDA exits. I don't get any details. The nvda_old.log file is not created.
No problem with the installed version.

### Expected behavior:
NVDA should start, no error.

#### NVDA installed/portable/running from source:
From source and portable copy.

#### NVDA version:
All

#### Windows version:
10 21H2

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Same behavior.

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by lukaszgo1** [comment]
Try starting with the debug logging by adding `--debug-logging` switch.

**Comment by aaclause** [comment]
No difference too. I forgot to specify that the NVDA startup sound is played, just before the Windows error sound.

```
D:\src\aaclause\nvda>runnvda.bat --debug-logging
Ensuring NVDA Python virtual environment
call py "D:\src\aaclause\nvda\\source\nvda.pyw" --debug-logging
Deactivating NVDA Python virtual environment

D:\src\aaclause\nvda>
```

**Comment by lukaszgo1** [comment]
I find it very odd that no log file is created. For source copies the log file is placed in the source directory - in your case that should be D:\src\aaclause\nvda\source\nvda.log. Could you double check that this file does not exist?

**Comment by CyrilleB79** [comment]
Maybe not related. But mentioning in case it is useful. Check your antivirus quarantine if any, specifically if the files were copied from another computer.

I have faced this issue when testing on my work's computer. A .dll file was quarantined by Symantec antivirus. In the log, I think that there was a critical error when trying to import from nvdaHelper.
